engine/class_modules/sc_mage.cpp

```diff
@@ -2620,6 +2596,26 @@ struct arcane_mage_spell_t : public mage_spell_t
     return 1.0 + p()->buffs.arcane_charge->check() * ( base + mastery );
   }
 
+  void execute() override
+  {
+    mage_spell_t::execute();
+
+    if ( p()->talents.intuition.ok() && !p()->buffs.intuition->check() && triggers.intuition )
+    {
+      constexpr int blp_threshold = 11;
+      constexpr double base_proc_chance = 0.01;
```

Comment:
Maybe add a comment here explaining why we're not using the spell data value, otherwise it looks ready to merge.